setup/security/authentication/index.md

```diff
@@ -3,6 +3,8 @@ layout: single
 title:  "Authentication"
 sidebar:
   nav: setup
+redirect_from: /setup/security/authentication/network-arch/
```

Comment:
What does this do? Network Architecture links should also be removed from this page

Reply:
That's in case anyone has a bookmark or direct link to that page (unlikely but possible).  User gets redirected to this page instead of a 404.